### PR TITLE
{data}[GCCcore/10.3.0] FreeTDS v1.3.3

### PR DESCRIPTION
**Pull request by scicomp-moffitt**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/f/FreeTDS/FreeTDS-1.3.3-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('Doxygen', '1.9.1'),
+]
+
+dependencies = [
+    ('libreadline', '8.1'),
+]
+
+# osdependencies = [OS_PKG_OPENSSL_DEV]
```
Comment (boegel) [inline easybuild/easyconfigs/f/FreeTDS/FreeTDS-1.3.3-GCCcore-10.3.0.eb:27]: @scicomp-moffitt Any reason this is commented out? Is OpenSSL required? If so, it should be added as a proper dependency now through the `OpenSSL` wrapper we have:

```python
dependencies = [
    ('libreadline', '8.1'),
    ('OpenSSL', '1.1', '', True),
]

Comment (verdurin) [reply easybuild/easyconfigs/f/FreeTDS/FreeTDS-1.3.3-GCCcore-10.3.0.eb:27]: @scicomp-moffitt any updates here?

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node362.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/3fe567bb89a3210a84eea21ae7edb552 for a full test report.

**Comment by smoors** [comment]
@scicomp-moffitt I've made changes to `sanity_check_paths` to make it more portable, and added `sanity_check_commands`.

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @scicomp-moffitt!

**Pull request merged**